packages/react-native/example-expo/App.tsx

```diff
@@ -0,0 +1,151 @@
+import { useState, useEffect, useRef } from "react"
+import { StatusBar } from "expo-status-bar"
+import { StyleSheet, Text, View, ScrollView, TouchableOpacity, SafeAreaView } from "react-native"
+import { getItemAsync, setItemAsync } from "expo-secure-store"
+import { Mnemonic } from "./mnemonic"
+import {
+    addEventListener,
+    connect,
+    defaultConfig,
+    getInfo,
+    LiquidNetwork,
+    listPayments,
+    removeEventListener,
+    prepareReceivePayment,
+    prepareSendPayment,
+    receivePayment,
+    sendPayment,
+    PaymentMethod
+} from "@breeztech/react-native-breez-sdk-liquid"
+import type { SdkEvent } from "@breeztech/react-native-breez-sdk-liquid"
+
+const MNEMONIC_STORE = "mnemonic"
+
+const DebugLine = ({ title, text }: { title: string; text?: string }) => {
+    return (
+        <TouchableOpacity style={{ flex: 1 }}>
+            <View style={{ margin: 5 }}>
+                <Text style={{ fontWeight: "bold" }}>{title}</Text>
+                {text && text.length > 0 ? <Text>{text}</Text> : <></>}
+            </View>
+        </TouchableOpacity>
+    )
+}
+
+const mnemonicGenerator = new Mnemonic()
+
+export default function App() {
+    const [lines, setLines] = useState<Array<{ at: number; title: string; text?: string }>>([])
+    const listenerIdRef = useRef<string>()
+
+    useEffect(() => {
+        const addLine = (title: string, text?: string): void => {
+            setLines((lines) => [{ at: new Date().getTime(), title, text }, ...lines])
+            console.log(`${title}${text && text.length > 0 ? ": " + text : ""}`)
+        }
+
+        const eventHandler = (event: SdkEvent) => {
+            addLine("SDK Event", JSON.stringify(event))
+        }
+
+        const bolt11Invoice: string = ""
+
+        const asyncFn = async () => {
+            try {
+                let mnemonic = await getItemAsync(MNEMONIC_STORE)
+                if (!mnemonic) {
+                    mnemonic = await mnemonicGenerator.generateMnemonic(256)
+                    await setItemAsync(MNEMONIC_STORE, mnemonic)
+                }
+
+                // Get API Key
+                const apiKey = process.env.EXPO_PUBLIC_API_KEY
```

Comment:
Can you rename this to `BREEZ_API_KEY`. I wasn't sure if I needed an Expo API key at first

Reply:
Ah yes, looks confusing. By the way, anything that starts with `EXPO_PUBLIC` in Expo will be also exposed during runtime. I'll rename it to EXPO_PUBLIC_BREEZ_API_KEY.